docs/tutorials/mastering.md

```diff
@@ -271,9 +1209,7 @@ Create a Juju model to represent the User Plane, using the cloud `user-plane-clu
 - `core-ip`: the IP address for the UPF to use on the `core` subnet
 - `gnb-subnet`: the subnet CIDR where the gNB radios are reachable.
 
-```{note}
-The `access-gateway-ip` is expected to forward the packets from the `access-interface` to the `gnb-subnet`
-```
+Run the commands on `juju-controller` VM.
```

Comment:
Generally speaking, avoid repeating the "run the command" sentence, instead write what exactly is being done (ex. `create the Juju overlay file on the Juju controller VM`)

Reply:
Done